tutorial/hero-image-placeholder.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Placeholder generation for the hero image
+parent: Malibu Tutorial
+nav_order: 27
+nav_exclude: true
+---
+
+# {{page.title}}
+
+_This tutorial was contributed by [Harshith](https://www.linkedin.com/in/harshith-raj-092ba4176)_
+
+An image placeholder is a background color over the space, on which an actual image will be loaded.
+
+How Quintype handles image placeholder?
+
+- Wrap `ResponsiveImage` component with `figure` tag.
+- Based on aspect ratio provide required `padding-top` to the `figure` tag using styleName/className.
+- Use a generic styleName/className to the `figure` tag to provide required `background- color`.
```

Comment:
*for the figure tag

Reply:
Formatted by grammerly

Reply:
Change to "Use a generic styleName/className for the `figure` tag to provide the required `background-color`."

Reply:
done